pkg/models/base.go

```diff
@@ -41,3 +41,19 @@ func (in *Base) DeepCopyInto(out *Base) {
 	out.CreatedAt = in.CreatedAt
 	out.UpdatedAt = in.UpdatedAt
 }
+
+func DropAll(db *gorm.DB) error {
+	result := db.Where("1=1").Delete(Rpm{})
+	if result.Error != nil {
+		return result.Error
+	}
+	result = db.Where("1=1").Delete(RepositoryConfiguration{})
+	if result.Error != nil {
+		return result.Error
+	}
+	result = db.Where("1=1").Delete(Repository{})
+	if result.Error != nil {
+		return result.Error
+	}
+	return nil
+}
```

Comment:
Wasn't sure exactly where to put this method, as you can't make functions public in _test files.   We could put it in seeds, but i think maybe we should move seeds into the model package in the long term.